cygnus-common/src/test/java/com/telefonica/iot/cygnus/backends/sql/SQLQueryUtilsTest.java

```diff
@@ -18,9 +18,9 @@
 
 package com.telefonica.iot.cygnus.backends.sql;
 
+import com.amazonaws.services.dynamodbv2.xspec.L;
```

Comment:
Should this new dependence be added to pom.xml ?

Reply:
Not really. Not sure why I added it, may be an auto suggestion from IntelliJ. Anyway, I removed it.

Fixed in [e0aa884](https://github.com/telefonicaid/fiware-cygnus/pull/1944/commits/e0aa8847b5596ace1a4bf45340f279f7aa8e90e0)